### DISCUSSION TITLE
TypeError: Cannot read property 'tagName' of null

### DISCUSSION THREAD

**Discussion by jw-miaem** [vercel/next.js#24380] [Help]
### What version of Next.js are you using?

10.0.3

### What version of Node.js are you using?

14.16.0

### What browser are you using?

chrome / ff

### What operating system are you using?

macOS

### How are you deploying your application?

Other platform

### Describe the Bug

Recently added a OneTrust cookie script. The script tag contains a data attribute called 'data-domain-script'. This seems to throw the type error so I'm guessing its not a valid type for the nextjs Head component?

![Screenshot 2021-04-22 at 1 04 42 pm](https://user-images.githubusercontent.com/778991/115845210-886dce00-a418-11eb-9d20-ed2dd956bf1c.png)

### Expected Behavior

to work with data attribute

### To Reproduce

Add a script tag inside of <Head /> component in _app.tsx. For example:

<script
              src="example-script.js"
              type="text/javascript"
              charSet="UTF-8"
              data-domain-script={`${exampleVariable}`}
            />

remove data attribute and all is fine

**Comment by apollonian** [comment]
> Add a script tag inside of component in _app.tsx. For example:

I'm able to add the snippet without any issues. Would it be possible for you to provide a reproducible CodeSandbox? Thanks!

<img width="987" alt="Screenshot 2021-04-23 at 9 55 36 PM" src="https://user-images.githubusercontent.com/2150306/115901335-a8d07380-a47e-11eb-9fca-71afdf542335.png">

**Reply by jw-miaem** [reply]
Thanks for this - jumping to conclusions, should have tried sandbox first - the variable is returning null as opposed to anything related to head component

**Reply by jw-miaem** [reply]
having said that it may be something else actually 👎 but not related to Head component from what I can tell
